### DISCUSSION TITLE
Can Mintpy process LICSAR data?

### DISCUSSION THREAD

**Discussion by zhouhuayun** [insarlab/MintPy#809] [General]
Hi all,
I can't find any document related to processing LICSAR. Does Mintpy support processing LICSAR data?

**Comment by scottstanie** [comment]
I believe LiCSBAS was made to process the LICSAR data https://github.com/yumorishita/LiCSBAS . Is there a specific feature that MintPy has not available in LiCSBAS that you wanted to use?

**Reply by zhouhuayun** [reply]
Thank you very much for your answer. I am currently trying to use LICSBAS.

**Comment by palatyle** [comment]
Sorry to bump this old thread, but I have the same question. I'd like to be able to atmospherically correct my timeseries and doing that via LICSBAS and GACOS seems like a major pain given the lack of API to use GACOS. I think it should be straightforward to just tell MintPy that it's dealing with an hyp3 stack of interferograms. The only wrinkle I foresee is the lack of look vectors. LICSAR provides N,E,U components of the LOS unit vector for each pixel, but I'm not sure if these can be projected to what hyp3 spits out (there's a lack of information of what conventions/units LICSAR is using). Would processing still work if look vectors weren't included? Any other issues that might crop up?

**Reply by yunjunz** [reply]
The e/n/u unit vector should be standard (as there should be no ambiguity). Given the [convention of incidence/azimuth angles defined in mintpy](https://github.com/insarlab/MintPy/blob/4f09c58629a47f70cbf0a66abaa736a15096386c/src/mintpy/utils/utils0.py#L566-L583), it should be straightforward to convert between them.

I have not tried LiCSAR products yet, if it's in lat/lon, and not in UTM, which is what hyp3 is using. Then using `hyp3` here will not work.

Anyway, the above two potential issues and metadata preparation should not be difficult to solve if one wants to give it a try.

**Reply by palatyle** [reply]
Cool, thanks Yunjun!
